# Glauconema phenotypic plasticity

![Version](https://img.shields.io/badge/Version-1.0-brightgreen)
![Release date](https://img.shields.io/badge/Release%20date-Jun.%222%2C%202024-brightgreen.svg)



## Contents

- [Introduction](#introduction)
- [System Requirement and dependency](#system-requirement-and-dependency)
- [Usage](#usage)
- [Dataset](#dataset)
- [Citation](#citation)

# Introduction

Glauconema phenotypic plasticity

# System Requirement and dependency

## Hardware Requirements

Glauconema phenotypic plasticity only requires a standard computer with sufficient RAM to support the operations defined by a user. For typical users, this would be a computer with about 2 GB of RAM. For optimal performance, we recommend a computer with the following specs:

  RAM: 8+ GB
  CPU: 4+ cores, 3.3+ GHz/core

## Software Requirements

R


# Usage
**a. Figure-1**

xxxxx
```
Figure_1L.R / R script Figure_1L.xlsx / input data for R
```

# dataset
Here we provide xx dataset .

[Synthetic Dataset 1](http://bioinfo.single-cell.cn/Released_Software/dynamic-meta-storms/data/synthetic_dataset_1.tar.gz) contains 40 synthetic metagenomes that are derived from 48 bacteria species.

[Synthetic Dataset 2](http://bioinfo.single-cell.cn/Released_Software/dynamic-meta-storms/data/synthetic_dataset_2.tar.gz) contains 100,000 synthetic metagenomes that are derived from 3,688 bacteria species.

[Real Dataset 1](http://bioinfo.single-cell.cn/Released_Software/dynamic-meta-storms/data/actual_dataset_1.tar.gz) contains 2,355 real human metagenomes produced by Human Microbiome Project Phase 1 (version 2).

[Real Dataset 2](http://bioinfo.single-cell.cn/Released_Software/dynamic-meta-storms/data/actual_dataset_2.tar.gz) contains 24 real environmental metagenomes produced by MetaSUB project.

[Source files](http://bioinfo.single-cell.cn/Released_Software/dynamic-meta-storms/data/metaphlan2.tree.tar.gz) of MetaPhlAn2 tree and taxonomy.

[Source files](http://bioinfo.single-cell.cn/Released_Software/dynamic-meta-storms/data/metaphlan3.tree.tar.gz) of MetaPhlAn3 tree and taxonomy.

# Citations

Pan et.al, Molecular basis of phenotypic plasticity in a marine ciliate Glauconema. 2024

# Glauconema phenotypic plasticity data & scripts

Pan et al. Molecular basis of phenotypic plasticity in a marine ciliate Glauconema. 2024

The Glauconema phenotypic plasticity data & scripts.zip file contains all secondary derived data and scripts of above paper.

In the zip file, there are multiple folders named with the figure number, like Glauconema_2024-Figure-1L, which contain the data and R scripts used by the drawing.

The each file are listedï¼š

Glauconema_2024-Figure-1L:		
Figure_1L.R	/	R script
Figure_1L.xlsx	/	input data for R
		
Glauconema_2024-Figure-1N-S6E:		
LHA0827_length_width.R	/	R script
LHA0827.xlsx	/	input data for R
LJL43_length_width.R	/	R script
LJL43.xlsx	/	input data for R
		
Glauconema_2024-Figure-1N-S6F:		
speed_LJL43_LHA0827_barplot.csv	/	input data for R
speed_LJL43_LHA0827_barplot.R	/	R script
		
Glauconema_2024-Figure-1O-S6G:		
LHA0827.R	/	R script
LHA0827.xlsx	/	input data for R
LJL43.R	/	R script
LJL43.xlsx	/	input data for R
		
Glauconema_2024-Figure-2A:		
ChrInfo.tab	/	input data for R for TBtools
geneid_pos.txt	/	input data for R for TBtools
illumima_depth.txt	/	input data for R for TBtools
LHA0827.GCratio	/	input data for R for TBtools
LHA0827.GCskew	/	input data for R for TBtools
LHA0827_gene.txt	/	input data for R for TBtools
LHA0827_length.txt	/	input data for R for TBtools
LinkRegion.tab	/	input data for R for TBtools
nano_depth.txt	/	input data for R for TBtools
OtherPara.tab	/	input data for R for TBtools
TrackInfo.tab	/	input data for R for TBtools
TrakFile.Cache	/	input data for R for TBtools
		
Glauconema_2024-Figure-2A/TrakFile.Cache:		
gla0827_gene.txt12029257	/	input data for R for TBtools
gla0827_gene.txt1255490	/	input data for R for TBtools
gla0827_gene.txt18025636	/	input data for R for TBtools
gla0827_gene.txt66293834	/	input data for R for TBtools
gla0827_gene.txt7978418	/	input data for R for TBtools
gla0827_gene.txt855281	/	input data for R for TBtools
gla0827_gene.txt95855326	/	input data for R for TBtools
gla0827_gene.txt96209234	/	input data for R for TBtools
illumima_depth.txt33455432	/	input data for R for TBtools
illumima_depth.txt61176885	/	input data for R for TBtools
illumima_depth.txt6283737	/	input data for R for TBtools
illumima_depth.txt91399328	/	input data for R for TBtools
illumima_depth.txt93359508	/	input data for R for TBtools
LHA0827.GCratio23293966	/	input data for R for TBtools
LHA0827.GCratio26603738	/	input data for R for TBtools
LHA0827.GCratio51187333	/	input data for R for TBtools
LHA0827.GCratio60245768	/	input data for R for TBtools
LHA0827.GCratio77036694	/	input data for R for TBtools
LHA0827.GCratio80722086	/	input data for R for TBtools
LHA0827.GCratio86495032	/	input data for R for TBtools
LHA0827.GCratio88197085	/	input data for R for TBtools
LHA0827.GCskew14575691	/	input data for R for TBtools
LHA0827.GCskew14738841	/	input data for R for TBtools
LHA0827.GCskew2382049	/	input data for R for TBtools
LHA0827.GCskew5128262	/	input data for R for TBtools
LHA0827.GCskew61911476	/	input data for R for TBtools
LHA0827.GCskew64570094	/	input data for R for TBtools
LHA0827.GCskew80035562	/	input data for R for TBtools
LHA0827.GCskew83818656	/	input data for R for TBtools
nano_depth.txt11769678	/	input data for R for TBtools
nano_depth.txt41836535	/	input data for R for TBtools
nano_depth.txt50542012	/	input data for R for TBtools
nano_depth.txt81665107	/	input data for R for TBtools
nano_depth.txt82142861	/	input data for R for TBtools
nano_depth.txt89599699	/	input data for R for TBtools
nano_depth.txt99706278	/	input data for R for TBtools
		
Glauconema_2024-Figure-2B-S6I:		
ontig_number_length.R	/	R script
data.csv	/	input data for R
		
Glauconema_2024-Figure-2C-S6J:		
gene_length_number.csv	/	input data for R
gene_length_number.R	/	R script
		
Glauconema_2024-Figure-2D-S6K:		
gene_per_contig_0827.csv	/	input data for R
gene_per_contig_0827.R	/	R script
gene_per_contig_43.csv	/	input data for R
gene_per_contig_43.R	/	R script
		
Glauconema_2024-Figure-2E-S6L:		
exon_per_gene_0827_43.csv	/	input data for R
exon_per_gene_0827_43.R	/	R script
		
Glauconema_2024-Figure-2F-S6M:		
intron_length.R	/	R script
intron_length.xlsx	/	input data for R
		
Glauconema_2024-Figure-3A:		
ChrInfo.tab	/	input data for R for TBtools
GenePos.tab	/	input data for R for TBtools
LinkRegion.tab	/	input data for R for TBtools
LJL43.GCratio	/	input data for R for TBtools
LJL43.GCskew	/	input data for R for TBtools
LJL43_gene.txt	/	input data for R for TBtools
LJL43_illu_depth.txt	/	input data for R for TBtools
LJL43_length.txt	/	input data for R for TBtools
LJL43_nano_depth.txt	/	input data for R for TBtools
LJL43.Nratio	/	input data for R for TBtools
OtherPara.tab	/	input data for R for TBtools
README.md	/	input data for R for TBtools
TrackInfo.tab	/	input data for R for TBtools
		
Glauconema_2024-Figure-3C:		
expand_genename.csv	/	input data for R
expand_genename_GO.csv	/	input data for R
expand_genename_GO.R	/	R script
		
Glauconema_2024-Figure-4:		
BP_database.csv	/	input data for R
CC_database.csv	/	input data for R
deseq_R.R	/	R script
F_C.csv	/	input data for R
F_C_deg.csv	/	input data for R
F_C_down_go_BP.csv	/	input data for R
F_C_down_go_CC.csv	/	input data for R
F_C_down_go_MF.csv	/	input data for R
F_C_downname.csv	/	input data for R
F_C_downname_KEGG.csv	/	input data for R
F_C_normalized.csv	/	input data for R
F_C_up_go_BP.csv	/	input data for R
F_C_up_go_CC.csv	/	input data for R
F_C_up_go_MF.csv	/	input data for R
F_C_upname.csv	/	input data for R
F_C_upname_KEGG.csv	/	input data for R
F_S.csv	/	input data for R
F_S_deg.csv	/	input data for R
F_S_down_go_BP.csv	/	input data for R
F_S_down_go_CC.csv	/	input data for R
F_S_down_go.csv	/	input data for R
F_S_down_go_MF.csv	/	input data for R
F_S_downname.csv	/	input data for R
F_S_downname_KEGG.csv	/	input data for R
F_S_GO.R	/	R script
F_S_KEGG.R	/	R script
F_S_up_go_BP.csv	/	input data for R
F_S_up_go_CC.csv	/	input data for R
F_S_up_go.csv	/	input data for R
F_S_up_go_MF.csv	/	input data for R
F_S_upname.csv	/	input data for R
F_S_upname_KEGG.csv	/	input data for R
gene2KO.csv	/	input data for R
gene_count_deg_F_S.csv	/	input data for R
gene_count_deg_F_S_down.csv	/	input data for R
gene_count_deg_F_S_up.csv	/	input data for R
gene_count_matrix.csv	/	input data for R
KO2ko.csv	/	input data for R
MF_database.csv	/	input data for R
name2ko.csv	/	input data for R
term2keggName.csv	/	input data for R
transcript_count_matrix.csv	/	input data for R
		
Glauconema_2024-Figure-5A:		
qPCR.pzfx	/	input data for R for GraphPad Prism 9
		
Glauconema_2024-Figure-5B:		
RNAi_largesystem.pzfx	/	input data for R for GraphPad Prism 9
		
Glauconema_2024-Figure-5C-5D:		
pgk1_RNA binding_qpcr.R	/	R script
qpcr-pgk1.xlsx	/	input data for R
qpcr-RNA binding.xlsx	/	input data for R
		
Glauconema_2024-Figure-5E-5F:		
pgk1.R	/	R script
pgk1.xlsx	/	input data for R
RNA binding.R	/	R script
RNA binding.xlsx	/	input data for R
		
Glauconema_2024-Figure-S2:		
data.txt	/	input data for R
Glauconema_percent.R	/	R script
observed_features_alpha_rarefaction.csv	/	input data for R
observed_otus_rarefaction_groups.R	/	R script
		
Glauconema_2024-Figure-S3:		
GC-LHA0827.R	/	R script
GC-LJL43.R	/	R script
LHA0827.csv	/	input data for R
LJL43.csv	/	input data for R
		
Glauconema_2024-Figure-S4:		
Figure S4.csv	/	input data for R
Figure S4.R	/	R script
		
Glauconema_2024-Figure-S6H:		
ChrInfo.tab	/	input data for R for TBtools
GenePos.tab	/	input data for R for TBtools
LinkRegion.tab	/	input data for R for TBtools
OtherPara.tab	/	input data for R for TBtools
TrackInfo.tab	/	input data for R for TBtools
		
length_width_ratios_P_value:		
LHA0827_LJL43_length_to_width_data_with_t_test_p_value.xlsx	/	input data for t.test 
		
swimming_speed_P_value:		
swimming_speed.R	/	R script
swimming_speed.xls	/	input data for R 

